src/aks-preview/azext_aks_preview/decorator.py

```diff
@@ -135,6 +141,59 @@ def init_nat_gateway_models(self) -> None:
         # for model_name, model_type in nat_gateway_models.items():
         #     setattr(self, model_name, model_type)
 
+    def _get_pod_identity_models(self) -> Dict:
```

Comment:
nit: why not just return a `SimpleNamespace` from here?

Reply:
No problem for me, but I am not sure if we need to keep a dictionary.

Reply:
I think we can simplify thing by returning a SimpleNamespace

Reply:
Updated (removed) in commit [#91c0ca3](https://github.com/Azure/azure-cli-extensions/pull/4243/commits/91c0ca36c0f84ef93033e1de3e98cf5596dfb32b).